### DISCUSSION TITLE
Simplified Profiling

### DISCUSSION THREAD

**Discussion by christianneureiter** [open-metadata/OpenMetadata#19012] [Ideas]
Not always the full profiling is required, sometimes it simply enough to get row counts to track the volume of a table over time. Can this be added to the profiling ingestion as an option?

**Comment by Prajwal214** [comment]
Yes, OpenMetadata allows you to configure the profiler to collect only specific metrics, such as row counts, without performing a full profiling. You can adjust the profiler settings at the table level to include or exclude specific columns and metrics. This can be done by enabling or disabling the column profile and selecting only the metrics you are interested in, such as row count [Docs](https://docs.open-metadata.org/latest/how-to-guides/data-quality-observability/profiler/workflow#4.-updating-profiler-setting-at-the-table-level).
